package.json

```diff
@@ -2,9 +2,14 @@
   "name": "matches-where-query",
   "version": "1.1.5",
   "main": "matchesWhereQuery.js",
+  "scripts": {
+    "test": "mocha tests/test.js --reporter spec"
```

Comment:
We should setup the CI to run this test, and also we can add more test to cover all cases.
Thx!

Reply:
Yes, thanks! Its working fine now and with the tests. :)